packages/editor-ui/src/views/SettingsApiView.vue

```diff
@@ -1,133 +1,119 @@
-		...mapStores(useRootStore, useSettingsStore, useUsersStore, useCloudPlanStore, useUIStore),
-		currentUser(): IUser | null {
-			return this.usersStore.currentUser;
-		},
-		isTrialing(): boolean {
-			return this.cloudPlanStore.userIsTrialing;
-		},
-		isLoadingCloudPlans(): boolean {
-			return this.cloudPlanStore.state.loadingPlan;
-		},
-		isPublicApiEnabled(): boolean {
-			return this.settingsStore.isPublicApiEnabled;
-		},
-		isRedactedApiKey(): boolean {
-			if (!this.apiKeys) return false;
-			return this.apiKeys[0].apiKey.includes('*');
-		},
-	},
-	methods: {
-		onUpgrade() {
-			void this.pageRedirectionHelper.goToUpgrade('settings-n8n-api', 'upgrade-api', 'redirect');
-		},
-		async showDeleteModal() {
-			const confirmed = await this.confirm(
-				this.$locale.baseText('settings.api.delete.description'),
-				this.$locale.baseText('settings.api.delete.title'),
-				{
-					confirmButtonText: this.$locale.baseText('settings.api.delete.button'),
-					cancelButtonText: this.$locale.baseText('generic.cancel'),
-				},
-			);
-			if (confirmed === MODAL_CONFIRM) {
-				await this.deleteApiKey();
-			}
-		},
-		async getApiKeys() {
-			try {
-				this.apiKeys = await this.settingsStore.getApiKeys();
-			} catch (error) {
-				this.showError(error, this.$locale.baseText('settings.api.view.error'));
-			} finally {
-				this.mounted = true;
-			}
-		},
-		async createApiKey() {
-			this.loading = true;
-
-			try {
-				const newApiKey = await this.settingsStore.createApiKey();
-				this.apiKeys.push(newApiKey);
-			} catch (error) {
-				this.showError(error, this.$locale.baseText('settings.api.create.error'));
-			} finally {
-				this.loading = false;
-				this.$telemetry.track('User clicked create API key button');
-			}
-		},
-		async deleteApiKey() {
-			try {
-				await this.settingsStore.deleteApiKey(this.apiKeys[0].id);
-				this.showMessage({
-					title: this.$locale.baseText('settings.api.delete.toast'),
-					type: 'success',
-				});
-				this.apiKeys = [];
-			} catch (error) {
-				this.showError(error, this.$locale.baseText('settings.api.delete.error'));
-			} finally {
-				this.$telemetry.track('User clicked delete API key button');
-			}
-		},
-		onCopy() {
-			this.$telemetry.track('User clicked copy API key button');
-		},
-	},
+const { showError, showMessage } = useToast();
+const { confirm } = useMessage();
+const { baseUrl } = useRootStore();
+const settingsStore = useSettingsStore();
+const cloudPlanStore = useCloudPlanStore();
+const documentTitle = useDocumentTitle();
+const i18n = useI18n();
+const { goToUpgrade } = usePageRedirectionHelper();
+
+const loading = ref(false);
+const mounted = ref(false);
+const apiKeys = ref<ApiKey[]>([]);
+const apiDocsURL = ref('');
+const telemetry = useTelemetry();
+
+const { isPublicApiEnabled, isSwaggerUIEnabled, publicApiPath, publicApiLatestVersion } =
+	settingsStore;
+
+const isTrialing = computed((): boolean => {
+	return cloudPlanStore.userIsTrialing;
+});
```

Comment:
nitpick: spaces between functions and maybe arrow functions instead. Also, keep order suggested in the document